orttraining/orttraining/core/graph/gradient_schema_defs.cc

```diff
@@ -70,6 +70,32 @@ void AddRepeatedOutputs(
   }
 }
 
+static void propagateRecvOutputTensorElemTypes(
+    InferenceContext& ctx,
+    const std::string& attributeName,
+    size_t outputSize) {
+  auto attr_proto = ctx.getAttribute(attributeName);
+  if (nullptr == attr_proto) {  // attribute not present
+      fail_type_inference("Value of attribute ", attributeName, " not specified");
```

Comment:
```suggestion
      fail_type_inference("Value of attribute ", attributeName, " not specified");
```
Can we print out the name of this operator in the error message?